scraping/spiders/mapahelp.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+import scrapy
+
+from config.constants import DEFAULT_CATEGORY
+
+
+log = logging.getLogger(__name__)
+
+
+CATEGORIES = {
+    "Finance": "Finance",
+    "Food": "Food",
+    "Health": "Medical",
+    "Housing": "Accommodation",
+    "Humanitarian Aid": "Food",
+    "Kennels": "Pets",
+    "Psychological Help": "Mental help",
+    "Shelter": "Accommodation",
+    "Sports": "Any help",
+    "Transfer": "Transport",
+    "Other Help": "Any help",
+}
+
+COUNTRIES = {
+    "Австралія": "Australia",
```

Comment:
Normally, you don't have to do this because our main data converter later would translate cities to English automatically. We don't do it for countries, but we could easily add it.

Reply:
Ah, okay, I'll pass it as it is then.